### PR TITLE
Fix Transmitting event

### PR DESCRIPTION
**Pull request by BoltonDev**
ev.IsAllowed now works and the event is no longer called every time, which is why IsTransmitting is Obsolete
Related to this https://github.com/Exiled-Team/EXILED/issues/1928

### CODE REVIEW

File: Exiled.Events/EventArgs/Player/TransmittingEventArgs.cs [hunk 1]
```diff
@@ -28,11 +31,15 @@ public class TransmittingEventArgs : IPlayerEvent, IDeniableEvent
...
-        public TransmittingEventArgs(Player player, VoiceModuleBase voiceModule)
+        /// <param name="isAllowed">
+        ///     <inheritdoc cref="IsAllowed" />
+        /// </param>
+        public TransmittingEventArgs(Player player, VoiceModuleBase voiceModule, bool isAllowed = true)
         {
             Player = player;
             VoiceModule = voiceModule;
-            IsTransmitting = voiceModule == null ? false : voiceModule.ServerIsSending && voiceModule.CurrentChannel == VoiceChatChannel.Radio;
+            IsTransmitting = true;
```
Comment (VALERA771) [inline Exiled.Events/EventArgs/Player/TransmittingEventArgs.cs:41]:
```suggestion
```

File: Exiled.Events/EventArgs/Player/TransmittingEventArgs.cs [hunk 2]
```diff
@@ -48,11 +55,12 @@ public TransmittingEventArgs(Player player, VoiceModuleBase voiceModule)
         /// <summary>
         ///     Gets a value indicating whether or not the player is transmitting.
         /// </summary>
+        [Obsolete("This is always true now.")]
         public bool IsTransmitting { get; }
```
Comment (VALERA771) [inline Exiled.Events/EventArgs/Player/TransmittingEventArgs.cs:59]: `IsTransmitting => true`

**Review by VALERA771 — COMMENTED** [review]
Make this and no more warnings

File: Exiled.Events/EventArgs/Player/TransmittingEventArgs.cs [hunk 3]
```diff
@@ -48,11 +49,11 @@ public TransmittingEventArgs(Player player, VoiceModuleBase voiceModule)
         /// <summary>
         ///     Gets a value indicating whether or not the player is transmitting.
         /// </summary>
-        public bool IsTransmitting { get; }
+        public bool IsTransmitting => true;
```
Comment (VALERA771) [inline Exiled.Events/EventArgs/Player/TransmittingEventArgs.cs:52]: I think obsolete was right here. Idk why you removed it

Comment (BoltonDev) [reply Exiled.Events/EventArgs/Player/TransmittingEventArgs.cs:55]: Idk i was probably tired :trollface:

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]
Fix conflicts

**Pull request merged**